src/test/java/reposense/analyzer/FileAnalyzerTest.java

```diff
@@ -23,7 +22,7 @@ public void allFileTest() {
         Assert.assertEquals(files.size(), 4);
         Assert.assertTrue(isFileExistence("annotationTest.java", files));
         Assert.assertTrue(isFileExistence("blameTest.java", files));
-        Assert.assertTrue(isFileExistence("newPos" + File.separator + "movedFile.java", files));
+        Assert.assertTrue(isFileExistence("newPos/movedFile.java", files));
```

Comment:
For the below function 

```
     private boolean isFileExistence(String filePath, List<FileInfo> files) {
         for (FileInfo file : files) {
             if (file.getPath().equals(filePath)) {
                 return true;
             }
         }
         return false;
     }
```

Instead of comparing strings, perhaps Paths will be better